mail/views.py

```diff
@@ -237,3 +238,37 @@ def post(self, request, *args, **kwargs):
             status=mailgun_response.status_code,
             data=generate_mailgun_response_json(mailgun_response)
         )
+
+
+class MailWebhookView(APIView):
+    """
+    View class that handles HTTP requests to capture bounced email
+    """
+    permission_classes = (MailGunWebHookPermission, )
+
+    def post(self, request, *args, **kargs):  # pylint: disable=unused-argument
+        """
+        POST method handler
+        """
+        event = request.POST.get("event", None)
+        recipient = request.POST.get("recipient", None)
+        error = request.POST.get("error", None)
+        message_headers = request.POST.get("message-headers", None)
+        log_error_on_bounce = request.POST.get("log_error_on_bounce", False)
```

Comment:
Please change this to

    log_error_on_bounce = request.POST.get("log_error_on_bounce", "")

This will ensure that it's always a string